bootstrap/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+# TestNet3, MainNet, RegTest
+btc.network=RegTest 
```

Comment:
I'd rather create 3 endpoints in 3 different modes. I think it's more convenient.

Reply:
Looks like BitcoinJ does not support multiple networks in one application instance
Here how to test
  @Test
    fun testBitcoinJWalletNetworks() {

        val wallet = Wallet(RegTestParams.get())

        val wallet2 = Wallet(TestNet3Params.get())
    }

Got:
java.lang.IllegalStateException: Context does not match implicit network params: org.bitcoinj.params.RegTestParams@2b55c76b vs org.bitcoinj.params.TestNet3Params@14ccf24b